src/components/Modal.js

```diff
@@ -22,39 +18,30 @@ const Modal = ( { onClose } ) => {
 		getBrand();
 	}, [] );
 
-	const [ refresh, setRefresh ] = useState( false );
+  const [refresh, setRefresh] = useState(false);
```

Comment:
Please format according to the master branch, I have added the linted and prettier configs and required dev packages in order to keep it up to date with wp js standards.